data/lang/ui-core/en.json

```diff
@@ -2027,6 +2039,14 @@
     "description": "Paintshop button",
     "message": "Reset Preview"
   },
+  "RESET_VIEW_ON_HYPERSPACE_EXIT": {
+    "description": "Reset View on Hyperspace Exit option",
+    "message": "Reset View on Hyperspace Exit"
+  },
+  "RESET_VIEW_ON_HYPERSPACE_EXIT_DESC": {
+    "description": "tooltip description in settings menu",
+    "message": "reset view to main game view when exiting from hyperspace"
```

Comment:
Leading word should be capitalized.

Reply:
Sure. But i used the surrounding definitions as a template to stay with the existing style.

do you have a coding guideline/style for new code? perhaps add pre-commit hooks?

Reply:
Fixed in fdb32c9e85790942aaa6a366b855a544751bb349

Reply:
In this area specifically, there's no enforceable style guide because the content of the strings are heavily context-based. In this case, it's just a matter of ensuring tooltip strings follow standard capitalization rules and are normalized with the rest of the interface (in this case, the string you copied from ideally also needs to be capitalized.)

We have/had a `clang-format` script to format C++ code; however, we've since disabled that as a pull-request check because `clang-format` has a very annoying habit of introducing opinionated formatting changes (or possibly bugs, who knows) that cannot be controlled via its own config.

Between that and the difficulty of keeping a sane config across 4+ simultaneous versions of clang-format (all of which throw errors if they see a config key from a later version of clang-format) it turned out to be significantly more trouble than it was worth.